keystone_api/main/settings.py

```diff
@@ -146,6 +146,13 @@
     'DEFAULT_SCHEMA_CLASS': 'drf_spectacular.openapi.AutoSchema',
 }
 
+SIMPLE_JWT = {
+    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=60)
```

Comment:
Expose this as an environmental variable and update the readme with a description for the new setting value.

Reply:
I think I set this more because it felt like the token was expiring while I was tinkering than because it really needs to be an hour. I don't have a great indication yet towards how long pulling the ResearchGroup info in from Slurm will take out on the cluster, so I'll just pull it for now.